Dockerfile

```diff
@@ -103,3 +114,44 @@ COPY --from=charts \
 COPY --from=crictl \
     /usr/local/bin/crictl \
     /bin/
+
+FROM build AS test
+ARG TARGETARCH
+VOLUME /var/lib/rancher/rke2
+VOLUME /var/lib/kubelet
+VOLUME /var/lib/cni
+VOLUME /var/log
+COPY bin/rke2 /bin/
+# use built air-gap images
+COPY build/rke2-airgap.tar /var/lib/rancher/rke2/agent/images/
+COPY build/images.txt /airgap-images.txt
+# use rke2 bundled binaries
+ENV PATH=/var/lib/rancher/rke2/bin:$PATH
+# for kubectl
+ENV KUBECONFIG=/etc/rancher/rke2/rke2.yaml
+# for crictl
+ENV CONTAINER_RUNTIME_ENDPOINT="unix:///run/k3s/containerd/containerd.sock"
+# for ctr
+RUN mkdir -p /run/containerd \
+    &&  ln -s /run/k3s/containerd/containerd.sock /run/containerd/containerd.sock
+# for go dns bug
+RUN mkdir -p /etc && \
+    echo 'hosts: files dns' > /etc/nsswitch.conf
+# for conformance testing
+RUN chmod 1777 /tmp
+RUN set -x \
+    && export DEBIAN_FRONTEND=noninteractive \
+    && apt-get -y update \
+    && apt-get -y install \
+    bash \
+    conntrack \
+    ebtables \
+    ethtool \
+    iptables \
+    jq \
+    less \
+    libseccomp2 \
+    socat \
+    vim
```

Comment:
Since #352 has been merged the base is now derived from Alpine therefore this will need to be updated to use apk.